### ISSUE TITLE
Improve the Validation Accuracy of the Model

### ISSUE THREAD

**Issue by SanjayMarreddi** [SanjayMarreddi/ML_Internship_Technocolabs#5]
You can try to improve the accuracy of the model by :
- Tuning the Hyper Parameters like :
     Learning Rate, Epochs, Batch Size, Number of Layers in CNN, Number of filters ,Size of filters, Value in Dropout Layers, Optimizers
- Using different Datasets.

**Comment by Jogesh-Mishra** [comment]
Can you please assign this issue to me ?

**Comment by sejalsksagar** [comment]
I am also interested in working on this issue. Is it possible to assign me too?

**Comment by SanjayMarreddi** [comment]
@Jogesh-Mishra @sejalsksagar I assigned to both of you. You can start working. Good Luck !!

**Comment by Jogesh-Mishra** [comment]
@SanjayMarreddi Thank You ! I will start working now.

**Comment by sejalsksagar** [comment]
@SanjayMarreddi Thank you! Starting to work on it.
